### PR TITLE
[pipeline] Add script to update nodedb & restart watchdog

### PR DESCRIPTION
**Pull request by janet-harmony**

### CODE REVIEW

File: pipeline/update_watchdog.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Assumptions:
+# experiment-deploy & nodedb repos are set up in the path
+# python3 installed & requests package is installed
+# github credentials for nodedb exist for current user
+# ssh watchdog is correctly configured
+
+base=`pwd`
```
Comment (AndyBoWu) [inline pipeline/update_watchdog.sh:9]: a better practice: https://stackoverflow.com/questions/192319/how-do-i-know-the-script-file-name-in-a-bash-script

File: pipeline/update_watchdog.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    w ) whoami="${OPTARG}" ;;
+    t ) target="${OPTARG}" ;;
+    u ) upload=true ;;
+    r ) restart=true ;;
+    ? ) help ;;
+  esac
+done
+
+# Update repo & reset
+if [[ ${upload} == true ]]; then
```
Comment (AndyBoWu) [inline pipeline/update_watchdog.sh:38]: "${upload}" might be better.

File: pipeline/update_watchdog.sh [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+# Push to master on nodedb
+if [[ ${upload} == true ]]; then
+  pushd ${nodedb}
+  git add ${target}/*
+  git commit -m "[update_watchdog] Updating ip lists for ${target}"
+  git push -f
+  popd
+  # Restart Watchdog
+  if [[ ${restart} == true ]]; then
+    ssh watchdog 'sudo sh -c "cd /home/jl/watchdog/nodedb && git stash && git pull -r"'
```
Comment (AndyBoWu) [inline pipeline/update_watchdog.sh:65]: create a variable for `/home/jl/watchdog/nodedb` and put it at the top?

File: pipeline/update_watchdog.sh [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    u ) upload=true ;;
+    r ) restart=true ;;
+    ? ) help ;;
+  esac
+done
+
+# Update repo & reset
+if [[ ${upload} == true ]]; then
+  pushd ${nodedb}
+  git reset --hard HEAD
```
Comment (LeoHChen) [inline pipeline/update_watchdog.sh:40]: git reset --hard origin/master
git clean -xdf

**Pull request merged**